src/software/ai/navigator/path_planner/theta_star_path_planner.h

```diff
@@ -61,22 +64,29 @@ class ThetaStarPathPlanner : public PathPlanner
     class CellHeuristic
     {
        public:
-        CellHeuristic() : parent_(0, 0), f_(0), g_(0), initialized_(false) {}
+        CellHeuristic()
+            : parent_(0, 0),
+              path_cost_and_end_dist_heuristic_(0),
+              best_path_cost_(0),
+              initialized_(false)
+        {
+        }
 
         /**
          * Updates CellHeuristics internal variables
          * Once updated, a CellHeuristic is considered intialized
          *
          * @param parent parent
-         * @param f f value
-         * @param g g value
+         * @param path_cost_and_end_dist_heuristic The path cost and end dist heuristic
+         * @param best_path_cost best_path_cost
          */
-        void update(Coordinate parent, double f, double g)
+        void update(Coordinate parent, double path_cost_and_end_dist_heuristic,
```

Comment:
nit: `parent` could be a const-reference

Reply:
fixed all these nits